From aa5bf816dbbc45b2243db0586ba8713073402007 Mon Sep 17 00:00:00 2001
From: Liu Gang <Gang.Liu@nxp.com>
Date: Mon, 11 Apr 2016 14:11:26 +0800
Subject: [PATCH 042/245] GPIO: Add GPIO kernel config into default config
 files

commit cf68f14f9fac30bb0bfe2d461d3481a1ab308db1
[doesn't apply arch/arm/configs/ls1021a_defconfig]
[doesn't apply arch/powerpc/configs/corenet32_fmanv3l_smp_defconfig]
[doesn't apply arch/powerpc/configs/corenet64_fmanv3h_smp_defconfig]
[doesn't apply arch/powerpc/configs/corenet64_fmanv3l_smp_defconfig]

For testing conveniently, add GPIO kernel config into
default config files for PowerPC and ARM platforms.

Signed-off-by: Liu Gang <Gang.Liu@nxp.com>
Integrated-by: Zhao Qiang <qiang.zhao@nxp.com>
---
 arch/arm64/configs/defconfig                 | 2 ++
 arch/powerpc/configs/corenet32_smp_defconfig | 1 +
 arch/powerpc/configs/corenet64_smp_defconfig | 1 +
 arch/powerpc/configs/mpc85xx_smp_defconfig   | 1 +
 4 files changed, 5 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index bfdc78c..5126c5d 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -114,6 +114,8 @@ CONFIG_SPI_PL022=y
 CONFIG_PINCTRL_MSM8916=y
 CONFIG_GPIO_PL061=y
 CONFIG_GPIO_XGENE=y
+CONFIG_GPIO_SYSFS=y
+CONFIG_GPIO_MPC8XXX=y
 CONFIG_POWER_RESET_XGENE=y
 CONFIG_POWER_RESET_SYSCON=y
 # CONFIG_HWMON is not set
diff --git a/arch/powerpc/configs/corenet32_smp_defconfig b/arch/powerpc/configs/corenet32_smp_defconfig
index 3765993..5414a8e 100644
--- a/arch/powerpc/configs/corenet32_smp_defconfig
+++ b/arch/powerpc/configs/corenet32_smp_defconfig
@@ -122,6 +122,7 @@ CONFIG_SPI=y
 CONFIG_SPI_GPIO=y
 CONFIG_SPI_FSL_SPI=y
 CONFIG_SPI_FSL_ESPI=y
+CONFIG_GPIO_SYSFS=y
 CONFIG_SENSORS_LM90=y
 CONFIG_SENSORS_INA2XX=y
 CONFIG_USB_HID=m
diff --git a/arch/powerpc/configs/corenet64_smp_defconfig b/arch/powerpc/configs/corenet64_smp_defconfig
index 33cd1df..f7aa84a 100644
--- a/arch/powerpc/configs/corenet64_smp_defconfig
+++ b/arch/powerpc/configs/corenet64_smp_defconfig
@@ -115,6 +115,7 @@ CONFIG_SPI=y
 CONFIG_SPI_GPIO=y
 CONFIG_SPI_FSL_SPI=y
 CONFIG_SPI_FSL_ESPI=y
+CONFIG_GPIO_SYSFS=y
 CONFIG_SENSORS_LM90=y
 CONFIG_SENSORS_INA2XX=y
 CONFIG_USB_HID=m
diff --git a/arch/powerpc/configs/mpc85xx_smp_defconfig b/arch/powerpc/configs/mpc85xx_smp_defconfig
index b6c7111..1bc0ec3 100644
--- a/arch/powerpc/configs/mpc85xx_smp_defconfig
+++ b/arch/powerpc/configs/mpc85xx_smp_defconfig
@@ -142,6 +142,7 @@ CONFIG_I2C_MPC=y
 CONFIG_SPI=y
 CONFIG_SPI_FSL_SPI=y
 CONFIG_SPI_FSL_ESPI=y
+CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_MPC8XXX=y
 CONFIG_SENSORS_LM90=y
 CONFIG_FB=y
-- 
2.1.0.27.g96db324

